website/content/commands/intention/delete.mdx

```diff
@@ -7,8 +7,18 @@ page_title: 'Commands: Intention Delete'
 
 Command: `consul intention delete`
 
+Corresponding HTTP API Endpoint: [\[DELETE\] /v1/connect/intentions/exact](https://www.consul.io/api-docs/connect/intentions#delete-intention-by-name)
```

Comment:
This actually has two corresponding HTTP API Endpoints. The other one is:
`[\[DELETE\] /v1/connect/intentions/:uuid](/api-docs/connect/intentions#delete-intention-by-id)`

Reply:
Fixed.